google-recaptcha.html

```diff
@@ -43,6 +49,18 @@
       _containerId:'',
       _captchaId:'',
 
+      _invalid: true,
+
+      behaviors: [
+        Polymer.IronFormElementBehavior,
+        Polymer.IronValidatableBehavior
+      ],
+
+      _getValidity: function () {
+        this.toggleClass('invalid', this._invalid);
+        return !this._invalid;
+      },
```

Comment:
@cbalit [wrote](https://github.com/GoogleWebComponents/google-recaptcha/issues/11#issuecomment-145783366):

> i wonder why you call toggle class in _getValidity method ?

`_getValidity` now gets called when you submit an iron-form that contains the element. Without this `toggleClass` call, the red border would not appear around an invalid recaptcha upon attempting to submit the form.